kafka-ui-react-app/src/components/KsqlDb/Query/QueryForm/QueryForm.tsx

```diff
@@ -43,13 +36,50 @@ const QueryForm: React.FC<Props> = ({
     formState: { errors },
   } = useForm<FormValues>({
     mode: 'onTouched',
-    resolver: yupResolver(validationSchema),
+    // resolver: yupResolver(validationSchema),
```

Comment:
?

Reply:
returned back validation but fields are not required for now, will add it when everything will work properly